cylc/flow/tui/util.py

```diff
@@ -31,14 +31,21 @@
 from cylc.flow.wallclock import get_unix_time_from_time_string
 
 
-def get_task_icon(status, is_held, start_time=None, mean_time=None):
+def get_task_icon(
+        status,
```

Comment:
maybe later we could force devs to name the params to avoid accidentally swapping `is_held` by `is_queued`, etc (i.e. `get_task_icon(status, *, is_held=False, ...)`